### PR TITLE
Sample of Get Element CSS Value

### PR DESCRIPTION
**Pull request by Peyton88**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [x] Change to the site (I am attaching a screenshot showing the before and after)
- [x] Code example added (and I also added the example to all translated languages)
- [x] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @Peyton88 ,

Thanks for the PR!

get Element css value actually returns a value of computed css style property of an element. we have WebDriver method like `getCssValue` in JS. The code sample you've provided is for `getAttribute`

i've updated doc with a code sample for JS. https://www.selenium.dev/documentation/fr/webdriver/web_element/#get-element-css-value

can you please update the code samples accordingly.

Thank you !

**Comment by Peyton88** [comment]
> Hi @Peyton88 ,
> 
> Thanks for the PR!
> 
> get Element css value actually returns a value of computed css style property of an element. we have WebDriver method like `getCssValue` in JS. The code sample you've provided is for `getAttribute`
> 
> i've updated doc with a code sample for JS. https://www.selenium.dev/documentation/fr/webdriver/web_element/#get-element-css-value
> 
> can you please update the code samples accordingly.
> 
> Thank you !

Thanks. I got a misunderstanding.
I've updated my commit.

File: docs_source_files/content/webdriver/web_element.en.md [hunk 1]
```diff
@@ -775,25 +775,57 @@ of an element in the current browsing context.
...
+
   {{< / code-panel >}}
   {{< code-panel language="ruby" >}}
-// Please provide a pr for the code sample
+
+# Navigate to Url
+driver.get 'https://www.example.com'
+
+# Retrieves the computed style property 'color' of linktext
+cssValue = driver.find_element(:link_text, 'More information...').css_value('color');
```
Comment (harsha509) [inline docs_source_files/content/webdriver/web_element.en.md]:
```suggestion
cssValue = driver.find_element(:link_text, 'More information...').css_value('color')
```

File: docs_source_files/content/webdriver/web_element.en.md [hunk 2]
```diff
@@ -775,25 +775,57 @@ of an element in the current browsing context.
...
+
 // Retrieves the computed style property 'color' of linktext
 let cssValue = await driver.findElement(By.linkText("More information...")).getCssValue('color');
+
     {{< / code-panel >}}
   {{< code-panel language="kotlin" >}}
-// Please provide a pr for the code sample
+
+// Navigate to Url
+driver.get('https://www.example.com');
```
Comment (harsha509) [inline docs_source_files/content/webdriver/web_element.en.md]:
```suggestion
driver.get('https://www.example.com')
```

File: docs_source_files/content/webdriver/web_element.en.md [hunk 3]
```diff
@@ -775,25 +775,57 @@ of an element in the current browsing context.
...
+
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-// Please provide a pr for the code sample
+
+# Navigate to Url
+driver.get('https://www.example.com');
+
+# Retrieves the computed style property 'color' of linktext
+cssValue = driver.findElement(By.LINK_TEXT, "More information...").value_of_css_property('color');
```
Comment (harsha509) [inline docs_source_files/content/webdriver/web_element.en.md]:
```suggestion
cssValue = driver.findElement(By.LINK_TEXT, "More information...").value_of_css_property('color')
```

File: docs_source_files/content/webdriver/web_element.en.md [hunk 4]
```diff
@@ -775,25 +775,57 @@ of an element in the current browsing context.
...
+
+// Retrieves the computed style property 'color' of linktext
+String cssValue = driver.findElement(By.linkText("More information...")).getCssValue('color');
+
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-// Please provide a pr for the code sample
+
+# Navigate to Url
+driver.get('https://www.example.com');
```
Comment (harsha509) [inline docs_source_files/content/webdriver/web_element.en.md]:
```suggestion
driver.get('https://www.example.com')
```

File: docs_source_files/content/webdriver/web_element.en.md [hunk 5]
```diff
@@ -775,25 +775,57 @@ of an element in the current browsing context.
...
+
     {{< / code-panel >}}
   {{< code-panel language="kotlin" >}}
-// Please provide a pr for the code sample
+
+// Navigate to Url
+driver.get('https://www.example.com');
+
+// Retrieves the computed style property 'color' of linktext
+val cssValue = driver.findElement(By.linkText("More information...")).getCssValue('color');
```
Comment (harsha509) [inline docs_source_files/content/webdriver/web_element.en.md]:
```suggestion
val cssValue = driver.findElement(By.linkText("More information...")).getCssValue('color')
```

**Review by harsha509 — CHANGES_REQUESTED** [review]
hi @Peyton88 ,

Thank you for making the changes. and apologies for reverting it, there are couple of things required to get this pr merged !

1. Can you please fill/complete the above checklist.
2. Semicolons are not required in few language bindings like kotlin, ruby, python can you please look into the suggested changes
3. Can you please make the same changes in other translated pages too like (es,fr,zh-cn).

Thank you!

**Comment by Peyton88** [comment]
> hi @Peyton88 ,
> 
> Thank you for making the changes. and apologies for reverting it, there are couple of things required to get this pr merged !
> 
> 1. Can you please fill/complete the above checklist.
> 2. Semicolons are not required in few language bindings like kotlin, ruby, python can you please look into the suggested changes
> 3. Can you please make the same changes in other translated pages too like (es,fr,zh-cn).
> 
> Thank you!

Hi @Harsha509 ,

Thanks for your kindly reminding.
Please see my updates.

**Review by harsha509 — APPROVED** [review]
Thank you @Peyton88 !

**Comment by harsha509** [comment]
deploy-preview looks good !

**Comment by harsha509** [comment]
Congratulations on your first contribution @Peyton88  🎉  🎉  🎉  🎉  🎉

**Comment by Peyton88** [comment]
> Congratulations on your first contribution @Peyton88 🎉 🎉 🎉 🎉 🎉

Thanks you~

**Pull request merged**